src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
This is a bit of a stop gap until we move to a standard venv recipe of some sort. (Assuming that we do. :-))

So I'm inclined to stick with this simple PR (with Jason's suggestion).

This issue illustrates why adding wheel support to eggs isn't much of a solution, at least unless the abi information was included in egg names and was used correctly. Not that that should be soooo hard, but not something I'm willing to volunteer for any time soon.